tests/test_flows.py

```diff
@@ -223,6 +234,14 @@ def init(self):
     def _check_log_internal(self, device: str, program: str, regex: str) -> bool:
         '''Search for regex in logfile for program on device.'''
         logfilename = os.path.join(self.rootdir, 'logs', device, 'beerocks_{}.log'.format(program))
+
+        # WSL doesn't support symlinks on NTFS, so resolve the symlink manually
```

Comment:
Actually it's a combination of issues. WSL doesn't use NTFS directly, it does implement a somewhat Linux compatible filesystem.
The problem is with Docker for Windows. There's a known bug that prevents the daemon (that actually runs on a Hyper-V VM) from mounting non-NTFS volumes into the containers.
Since we map the `build` and `tmp` folders into the `prplmesh-runner`, all the logs are eventually created on an NTFS filesystem that doesn't support symlinks.

Reply:
Ok, thanks for the explanation :-)

Reply:
Actually, NTFS does support symlinks (since NTFS version 3.0, which was released together with Windows 2000). Windows API support (e.g. OpenFile) for symlinks was added to Windows in Windows Vista, though linux (using ntfs-3g) could create these symlinks on NTFS even before then.
You can create those symlinks on Windows with the `mklink` command.
Annoyingly, IIRC you always need administrator privileges to create symlinks on windows though.

Reply:
Actually, it seems my information is outdated: https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/

Reply:
Symlinks work fine when created from Windows using `mklink`, but when created in WSL, they appear as regular files on NTFS.